scripts/wrappers/version.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+
+import json
+from os import environ
+from pathlib import Path
+
+
+def get_upstream_versions() -> str:
+    versions_file = Path(environ["SNAP"]) / "versions.json"
+    with open(versions_file, mode="r") as file:
+        versions = json.loads(file.read())
+        return versions
+
+
+def get_snap_version() -> str:
+    return environ["SNAP_VERSION"]
```

Comment:
```suggestion
    return os.getenv("SNAP_VERSION", "(unknown)")
```

Reply:
isn't `SNAP_VERSION` a variable that we expect to be there always?